Tasks/AzureRmWebAppDeployment/AzureRmWebAppDeployment.ts

```diff
@@ -18,8 +18,8 @@ async function run() {
 		var deployToSlotFlag: boolean = tl.getBoolInput('DeployToSlotFlag');
 		var resourceGroupName: string = tl.getInput('ResourceGroupName');
 		var slotName: string = tl.getInput('SlotName');
-		var package: string = tl.getPathInput('Package');
-		var setParametersFile: string = tl.getPathInput('SetParametersFile');
+		var webDeployPkg: string = trimDoubleQuotes(tl.getPathInput('Package'));
```

Comment:
Next sprint we are going to add support for cross plat app deployments then those apps will be deployed using cross plat technology.  Now, it's used only in windows. This task is not merged with master until next sprint.